scipy/stats/tests/test_stats.py

```diff
@@ -1771,6 +1771,30 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[0], z0_expected)
         assert_array_almost_equal(z[1], z1_expected)
 
+    def test_zscore_nan_propagate(self):
+        # Test nan_policy set to 'propogate'
```

Comment:
You could just leave out the comment here as the method name is already fairly descriptive. (Similarly below.)

If you decide to keep it, note that "propagate" is misspelled.

Reply:
I will remove the comments because I agree that they are not necessary.